rpmbuild/bin/copr-rpmbuild-log

```diff
@@ -48,7 +48,7 @@ def _tail_log(config):
 
     for fname in [pidfile, live_log]:
         if not os.path.exists(fname):
-            LOG.warning("File %s doesn't exist, yet", fname)
+            print("File {0} doesn't exist, yet".format(fname))
```

Comment:
But... is this print actually visible somewhere?

Reply:
It was when I was debugging the script in the console but I am not sure if it will appear in some of the backend logs. Either way, I don't know if there is anything else we can do. We obviously cannot log, that logging doesn't work yet :smile: